### DISCUSSION TITLE
using piper with android?

### DISCUSSION THREAD

**Discussion by MarkoSchuetz** [rhasspy/piper#236] [Q&A]
I'm using piper to learn Chinese pronunciation: I've configured it as one of the program sources in GoldenDict and I use it to pipe a selection of 汉字  so I can get a first idea of how the pronounce it. But I often find myself on Android and would like to use piper there also. Has anyone already worked towards this?

**Comment by csukuangfj** [comment]
Please see https://github.com/rhasspy/piper/issues/257

It supports Android.

---

You can download pre-built Android APKs at
https://k2-fsa.github.io/sherpa/onnx/tts/apk-engine.html
